index 11743bbb8f..32f7642468 100644
--- a/build_library/grub.cfg
+++ b/build_library/grub.cfg
@@ -79,7 +79,7 @@ if [ -z "$linux_console" ]; then
         terminal_output console serial_com0
     elif [ "$grub_platform" = efi ]; then
         if [ "$grub_cpu" = arm64 ]; then
-            set linux_console="console=ttyAMA0,115200n8"
+            set linux_console="console=ttyAMA0,115200n8 console=tty0"
         else
             set linux_console="console=ttyS0,115200n8 console=tty0"
        fi
